language-api/src/main/java/de/jplag/Language.java

```diff
@@ -123,4 +123,18 @@ default boolean supportsNormalization() {
     default boolean requiresCoreNormalization() {
         return true;
     }
+
+    /**
+     * @return True, if the language module can be used by the multi-language module
+     */
+    default boolean supportsMultilanguage() {
+        return true;
+    }
+
+    /**
+     * @return True, if the language module should be used in case of ambiguities
```

Comment:
Should be clearer, e.g. 
```suggestion
     * @return True if the language module should be prioritized in the multi-language case, if multiple language modules support the same file extension.
```